.github/workflows/automerge.yml

```diff
@@ -0,0 +1,23 @@
+name: automerge
+on:
+  pull_request:
+    types:
+      - labeled
+      - edited
+      - synchronized
+  status: {}
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: automerge
+        uses: "pascalgn/automerge-action@ccae530ae13b6af67a7a2009c266fe925844e658"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          MERGE_LABELS: "Ready to merge, !work in progress, !needs discussion, !needs more info, !needs discovery, !needs code updates, !needs code review, !needs changelog, !needs browser testing, !needs branch testing, !Needs AT updates, !merge conflicts"
+          MERGE_REMOVE_LABELS: "Ready to merge"
+          MERGE_METHOD: "squash"
+          MERGE_COMMIT_MESSAGE: "Auto merge {pullRequest.number}"
```

Comment:
I haven't used Actions before so I don't know what some of these properties mean.

I'm particularly curious about `MERGE_COMMIT_MESSAGE` and how it might apply to our conventional commits practice.  There are two scenarios I can imagine:

 1. Since `MERGE_METHOD` is `"squash"`, `MERGE_COMMIT_MESSAGE` becomes the message in the squashed commit
 2. `MERGE_COMMIT_MESSAGE` is the commit message for (literal) "merge commits" which only take place when a non-fast-forward merge happens

Since `UPDATE_METHOD` is set to `"rebase"`, there should never be a merge commit because master will get rebased under every PR's branch before it gets merged into master, and due to that rebase, every merge should be a fast-forward and thus there should never be a merge commit.

tl;dr: will this impact our conventional commit practice?